server/package.json

```diff
@@ -31,6 +30,7 @@
     "@nestjs/typeorm": "^6.1.3",
     "@types/uuid": "^3.4.5",
     "apollo-server-express": "^2.6.9",
+    "auth-utils": "../lib/auth-utils",
```

Comment:
`@libero` prefix to make it clear we're not using the authored `auth-utils` package on npm

Reply:
@Cooryd I tried and got this error:

```
[1/4] Resolving packages...
error Couldn't find any versions for "auth-utils" that matches "@libero/auth-utils"
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Service 'server_npm' failed to build: The command '/bin/sh -c yarn' returned a non-zero code: 1
make[1]: *** [install_server_packages] Error 1
make[1]: Leaving directory `/home/travis/build/libero/reviewer'
make: *** [server_ci] Error 2
```
I suggest (because this is not related to my change) that if we need to fix this you can raise another ticket. /cc @petereast